pxb-tag/README.md

```diff
@@ -0,0 +1,33 @@
+# @pxblue/tag
+
+This Command Line tool is a utility for automatically creating tags and github releases through a continuous integration pipeline.
+Given a `package.json` and a `CHANGELOG.md`, it will parse out the latest release notes and publish the latest release.
+
+## Prerequisites
+
+In order to use this utility you must have the following installed:
+
+-   [NodeJS](https://nodejs.org/en/download/) 12+
+-   npm
+-   [Github CLI](https://cli.github.com/)
+
+## Usage
+
+You can use this package by running it in the root directory of your project with npx (recommended):
+
+```
+npx -p @pxblue/publish pxb-publish -b <branch-name>
```

Comment:
this should be pxb-tag right?

Reply:
Yes, and I forgot to include the tag suffix flag as well.